### PR TITLE
[3.3] Replace ResourceManager with PathResolver in the configuration validators

### PR DESCRIPTION
**Pull request by GawainLynch**

### CODE REVIEW

File: src/Configuration/Validation/Apache.php [hunk 1]
```diff
@@ -31,13 +31,11 @@ public function check()
...
         $isApache = strpos($serverSoftware, 'Apache') !== false;
-        /** @var LowlevelChecks $verifier */
-        $verifier = $this->resourceManager->getVerifier();
-        if ($verifier && $verifier->disableApacheChecks === true || !$isApache) {
+        if (!$isApache) {
             return;
         }
 
-        $path = $this->resourceManager->getPath('web/.htaccess');
+        $path = $this->pathResolver->resolve('web/.htaccess');
```
Comment (CarsonF) [inline src/Configuration/Validation/Apache.php]: `%web%/.htaccess`

File: src/Configuration/Validation/Database.php [hunk 2]
```diff
@@ -109,7 +109,7 @@ protected function doDatabaseSqliteCheck(array $dbConfig)
             // At this point, it is possible that the site has been moved and
             // the configured Sqlite database file path is no longer relevant
             // to the site's root path
-            $cacheJson = $this->resourceManager->getPath('cache/config-cache.json');
+            $cacheJson = $this->pathResolver->resolve('cache/config-cache.json');
```
Comment (CarsonF) [inline src/Configuration/Validation/Database.php]: `%cache%/config-cache.json`

File: tests/phpunit/unit/Configuration/Validation/DatabaseSqliteTest.php [hunk 3]
```diff
@@ -187,7 +187,7 @@ public function testSqliteFileNotExistsDirectoryWritableCacheReset()
         ];
         $this->config->get('general/database')->willReturn($databaseConfig);
         $this->config->initialize()->shouldBeCalled();
-        $this->resourceManager->getPath('cache/config-cache.json')->shouldBeCalled();
+        $this->pathResolver->resolve('cache/config-cache.json')->shouldBeCalled();
```
Comment (CarsonF) [inline tests/phpunit/unit/Configuration/Validation/DatabaseSqliteTest.php]: `%cache%/config-cache.json`

**Comment by GawainLynch** [comment]
> Can you confirm that path resolver isn't invoked until boot?

https://github.com/bolt/bolt/blob/release/3.3/src/EventListener/ConfigListener.php#L91

**Comment by bobdenotter** [comment]
Now that AWS seems to be back, I restarted travis.

**Pull request merged**